migrations/2023-03-14-123541_add_cards_info_table/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Your SQL goes here
+CREATE TABLE cards_info (
+    card_iin VARCHAR(16) PRIMARY KEY,
+    card_issuer TEXT,
+    card_network TEXT,
+    card_type TEXT,
+    card_subtype TEXT,
+    card_issuing_country TEXT,
+    bank_code_id VARCHAR(32),
+    bank_code VARCHAR(32),
+    country_code VARCHAR(32),
+    date_created TIMESTAMP NOT NULL,
+    last_updated TIMESTAMP,
+    last_updated_provider TEXT
```

Comment:
Are these 3 columns being used anywhere?

Reply:
These are used internally, to keep track of when the card bin was created, updated and the provider which updated this card bin